components/Inputs/inputs.css

```diff
@@ -0,0 +1,303 @@
+@import url("../../Utilities/utilities.css");
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.chip-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 2rem 0;
+  border-radius: 0.5rem;
+  margin-top: 5rem;
+  width: 50rem;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: aliceblue;
+}
+
+.all-el {
+  display: flex;
+}
+
+.all-items {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  padding: 0;
+  font-size: 0;
+}
+
+.chip-items {
+  border: none;
+  background-color: var(--light-cyan);
+  border-radius: 3rem;
+  list-style: none;
+  padding: 0.3rem;
+  margin: 0.2rem 0.7rem;
+  font-size: 1.4rem;
+}
+
+.inp {
+  border: none;
+  margin-left: 0.5rem;
+  outline: none;
+  height: 3rem;
+  font-size: 1.5rem;
+  width: 10rem;
+}
+
+.fa-dot-circle {
+  font-size: 1rem;
+  margin-right: 1rem;
+}
+
+.hide-chips {
+  display: none;
+}
+
+.fa-times {
+  font-size: 1rem;
+  padding: 0 0.4rem;
+  cursor: pointer;
+}
+
+.chip-items:hover {
+  background-color: var(--light-cyan);
+}
+
+.fa-times:hover {
+  color: var(--brown);
+}
+
+/* Radio button */
+.Phone-ringtone {
+  display: flex;
+  width: 15rem;
+  height: 20.5rem;
+  flex-direction: column;
+  margin-top: 0.5rem;
+  background-color: var(--white);
+  padding: 0.5rem;
+  border-radius: 0.2rem;
+  margin-left: 2rem;
+  box-shadow: 0px 0px 5px 1px var(--black);
+}
+
+.all-ring-list {
+  overflow-y: scroll;
+  margin: 0.5rem 0rem;
+  display: flex;
+  flex-direction: column;
+  height: 14.5rem;
+  color: var(--blue-alert);
+}
+
+li p {
+  margin-left: 2rem;
+}
+
+.ring-cta {
+  margin-left: auto;
+  border-top: 0.9px solid var(--black);
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 0.8rem;
+  padding-top: 0.5rem;
+}
+
+.chosenRingtone {
```

Comment:
your naming conventions should be consistent.

Reply:
Will do that